src/app/shared/event-log/event-log-table.component.spec.ts

```diff
@@ -93,7 +93,7 @@ describe('EventLogTableComponent', () => {
     let firstEvent = EVENTS[0];
 
     expect(firstRowCells[COL_EVENT].nativeElement.textContent).toBe(firstEvent.event_name);
-    expect(firstRowCells[COL_DATE].nativeElement.textContent).toBe('10 May 2017, 10:00:00 AM');
+    expect(firstRowCells[COL_DATE].nativeElement.textContent).toBe('10 May 2017, 10:00:00 AM UTC: 10 May 2017, 10:00:00 AM');
```

Comment:
Why are we changing this?

Reply:
@MKaramath added a local date modifier and a tooltip for the event details component, however this table seems to have been forgotten. Which leads different time displayed in log table and detail. This is just for consistency.

Reply:
👍